src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { IPoint } from '@uifabric/utilities';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { ContextualMenu, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import * as React from 'react';
+import { kebabMenuButton } from './card-footer.scss';
+
+export interface CardKebabMenuButtonState {
+    isContextMenuVisible: boolean;
+    target?: HTMLElement | string | MouseEvent | IPoint | null;
```

Comment:
why is target optional?

Reply:
I don't think so, this is the pattern we follow for all drop downs